Automattic/wp-calypso
# Notifications: Relative Sizing for Header Elements

#### Changes proposed in this Pull Request
* I initially took a mathematical approach. p tags on the browser are normally 16px, but on WordPress, it is 14px. I took the size of the headers that the browser set by default, and then reduced them at the same rate. I only applied these to h1 to h3, since by some competitive research, h4-h6 are ignored.
* I got this linting error:

```
36:14  ✖  Expected "2.742rem" to be one of "0.75, 0.875, 1, 1.25, 1.5, 2, 2.25, 3, 3.375" with unit "rem"   scales/font-size
 40:14  ✖  Expected "2.057rem" to be one of "0.75, 0.875, 1, 1.25, 1.5, 2, 2.25, 3, 3.375" with unit "rem"   scales/font-size
 44:14  ✖  Expected "1.542rem" to be one of "0.75, 0.875, 1, 1.25, 1.5, 2, 2.25, 3, 3.375" with unit "rem"   scales/font-size
```

So I tried each bounds of constraints that were closer to my calculations.

Option A | Option B
--- | ---
<img width="390" alt="Screen Shot 2020-10-06 at 6 56 07 PM" src="https://user-images.githubusercontent.com/66652282/95268993-7de64100-0806-11eb-9a32-482690e0b6c3.png"> | <img width="392" alt="Screen Shot 2020-10-06 at 6 57 20 PM" src="https://user-images.githubusercontent.com/66652282/95269004-83438b80-0806-11eb-8161-ac7786652417.png">

The latter seems more reasonable in this case, given the constraints and relative to the other text sizes, so I chose Option B.

#### Testing instructions
1. Checkout this branch
2. See the"Creating post notification" section in README at apps/notifications/src/doc/note-rendering.md
3. Use the "Follow" button that appears in the bottom right corner of the blog to add it to your Reader subscriptions. Then navigate to https://wordpress.com/following/manage and enable "Notify me of new posts" for the blog. With this enabled, future posts on that blog will appear as notifications in your masterbar.
4. Create a post for the blog you're following with the header tags h6, h5, h4, h3, h2 and h1.
5. Go to your masterbar and select the notification that contains the styling. Confirm the headers are styled with relative sizing (h1 being the largest, h6 being the smallest)

Before | After
--- | ---
<img width="396" alt="Screen Shot 2020-10-06 at 5 20 18 PM" src="https://user-images.githubusercontent.com/66652282/95264710-d2d18980-07fd-11eb-97d8-e2f729b9d19a.png"> | <img width="392" alt="Screen Shot 2020-10-06 at 6 57 20 PM" src="https://user-images.githubusercontent.com/66652282/95268826-13cd9c00-0806-11eb-8deb-c0b7c3b9fdf1.png">


Fixes #46111
